File: DeviantArtFs/Types/ResponseTypes/StashDeltaEntry.fs
namespace DeviantArtFs

type StashDeltaEntry = {
    itemid: int64 option
    stackid: int64 option
    metadata: StashMetadata option
    position: int option
} with
    member this.GetItemId() = OptUtils.longDefault this.itemid
    member this.GetStackId() = OptUtils.longDefault this.stackid
    member this.GetPosition() = OptUtils.intDefault this.position
    member this.GetMetadata() = OptUtils.recordDefault this.metadata

File: DeviantArtFs.Types/StashDeltaEntry.cs
using System;

namespace DeviantArtFs.Types {
    public class StashDeltaEntry {
        public long? itemid { get; set; }
        public long? stackid { get; set; }
        public StashMetadata? metadata { get; set; }
        public int? position { get; set; }
    }
}
